### PR TITLE
feat: header redesign

### PR DESCRIPTION
**Pull request by narekpetrosyan**
## Description

- Redesign the **Header** component for both connected/not connected cases
- Remove **children** prop from Header component
- Add **TapBar** for mobile view 
- Remove **Sidebar** from the screen
- Update **.gitattributes** file

## Checklist

- [ ] Requires dependency update?
- [x] Manually check the changes visually
- [x] Verify that the code compiles without errors or warnings
- [x] Looks good on large screens
- [x] Looks good on mobile
- [x] Confirm that no existing functionality has been broken by the changes
- [ ] Ensure that the pull request is small and consistent with one main task per pull request
- [x] Tag reviewers in the pull request

Closes [TASK-ID: #8692h5a5p](https://app.clickup.com/t/8692h5a5p)

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -121,7 +121,7 @@ export function shortenAddress(address: string | null | undefined, chars = 4): s
     return ""
   }
 
-  return `${address.substring(0, chars + 2)}...${address.substring(42 - chars)}`
```
Comment (edward-arinin-web-dev) [inline src/utils/index.ts:124]: Do not change core functionalities

**Review by edward-arinin-web-dev — APPROVED** [review]
1. fix broken font-family at comment section, placeholder: "add comment" https://localhost:3000/dao/0xbb807205a0dc94d3268b59be6dd2466c1ef0ba79/dao-proposals/off-chain/139/details
feat_ Redesign Header component by narekpetrosyan · Pull Request #489 · dexe-network_investing-dashboard - Google Chrome 25 07 2023 19_29_19

2. fix proposal tabs padding at https://localhost:3000/dao/0xbb807205a0dc94d3268b59be6dd2466c1ef0ba79/about
feat_ Redesign Header component by narekpetrosyan · Pull Request #489 · dexe-network_investing-dashboard - Google Chrome 25 07 2023 19_28_04

**Review by edward-arinin-web-dev — CHANGES_REQUESTED** [review]
fix issues described above before merge

File: src/utils/index.ts (hunk 1 above)
Comment (narekpetrosyan) [reply src/utils/index.ts:124]: Reversed.

**Review by edward-arinin-web-dev — APPROVED** [review]

**Pull request merged**